recharts/recharts
# chore: upgrade react-resize-detector to 8.x

## Description


#3410  upgrade to latest resize detector

## Related Issue





#3410 

## Motivation and Context


upgrade to latest resize detector
see https://github.com/maslianok/react-resize-detector/releases/tag/v8.0.0

## How Has This Been Tested?




- run demos
- run tests
- run storybook
- check console for resize-detector related errors

Look at breaking changes here https://github.com/maslianok/react-resize-detector/releases/tag/v8.0.0

Breaking changes called out in changenotes:
* Generated TS types inside the build folder moved to the types folder - doesn't break us
* Remove the deprecated withPolyfill.d.ts file - not supported by us
* ComponentsProps => ResizeDetectorProps - doesn't break us
* patchResizeHandlerType => PatchedResizeObserverCallback - doesn't break us
* If both handleWidth and handleHeight are turned off, ResizeObserver and resizeCallback won't be created - these are always created for us

## Screenshots (if appropriate):

## Types of changes




## Checklist:
